Core.ApplicationServices/GDPR/DataProcessingRegistrationApplicationService.cs

```diff
@@ -170,6 +170,16 @@ public Maybe<OperationError> ValidateSuggestedNewRegistrationName(int organizati
                     .Match(result => result, () => new OperationError("Invalid reference Id", OperationFailure.BadInput)));
         }
 
+        public Result<ExternalReference, OperationError> ClearMasterReference(int id)
```

Comment:
Det er en skidt kontrakt for den kan jo godt være null. Jeg ville nok vælge `Maybe<OperationError>` i stedet for på interface niveau. eller `Result<Maybe<ExternalReference>, OperationError>` for at angive at det kun er måske at referencen var der